purchase_last_price_info/models/product_product.py

```diff
@@ -0,0 +1,84 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+# Copyright 2019 ForgeFlow S.L.
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    last_purchase_line_ids = fields.One2many(
+        comodel_name="purchase.order.line",
+        inverse_name="product_id",
+        domain=[("state", "in", ["purchase", "done"])],
+        string="Last Purchase Order Lines",
+    )
+    last_purchase_line_id = fields.Many2one(
+        comodel_name="purchase.order.line",
+        compute="_compute_last_purchase_line_id",
+        string="Last Purchase Line",
+        store=True,
```

Comment:
Why is this stored? This is going to trigger a huge computation when installing the module.